vm/systemSmartContracts/governance.go

```diff
@@ -1036,7 +1047,7 @@ func (g *governanceContract) computeEndResults(currentEpoch uint64, proposal *Ge
 	voteStarted := currentEpoch >= proposal.StartVoteEpoch
 	if g.enableEpochsHandler.IsFlagEnabled(common.GovernanceFixesFlag) && !voteStarted {
 		g.eei.Finish([]byte("Proposal closed before voting started"))
-		return true
+		return false
```

Comment:
this now means that the proposal fails when closing it before vote started and the proposer looses the LostProposalFee, right?